### PR TITLE
Feat/authenticate not found updated at

### PR DESCRIPTION
**Pull request by defra-dom**

### CODE REVIEW

File: app/graphql/types/customer/customer.gql [hunk 1]
```diff
@@ -103,6 +103,16 @@ type CustomerAuthenticationQuestions {
   The memorable place question.
   """
   memorablePlace: String
+
+  """
+  The date the record was last updated.
+  """
+  updatedAt: String
```
Comment (jack-mccabe-defra) [inline app/graphql/types/customer/customer.gql:110]: Shouldn't this be a timestamp type?

Comment (defra-dom) [reply app/graphql/types/customer/customer.gql:110]: sure ok ill sort this

File: test/integration/graphql/customer.test.js [hunk 2]
```diff
@@ -110,6 +113,90 @@ describe('Query.customer', () => {
...
+    })
+
+    expect(result).toEqual({
+      data: {
+        customer: {
+          authenticationQuestions: {
+            memorableDate: null,
+            memorableEvent: null,
+            memorablePlace: 'some location',
+            updatedAt: 'some date',
```
Comment (jack-mccabe-defra) [inline test/integration/graphql/customer.test.js:193]: This doesn't look right?

**Review by jack-mccabe-defra — APPROVED** [review]
LGTM

**Pull request merged**